core/sqf/src/tm/tmddlrequests.cpp

```diff
@@ -40,14 +40,24 @@ JNIEXPORT jint JNICALL Java_org_apache_hadoop_hbase_client_transactional_RMInter
    char la_tbldesc[TM_MAX_DDLREQUEST_STRING];
    char la_tblname[TM_MAX_DDLREQUEST_STRING];
    char* str_key;
-   str_key = new char[TM_MAX_DDLREQUEST_STRING];
    char** la_keys;
    la_keys = new char *[TM_MAX_DDLREQUEST_STRING];
    int lv_error = FEOK;
 
+   if( pv_keyLength > TM_MAX_DDLREQUEST_STRING)
+   {
+     cout << "Table key length is larger than max allowed " << pv_keyLength << endl;
```

Comment:
A native English speaker will see "Table key length is larger than max allowed 5123" and think that 5123 is the max, instead of the actual key length value. It might be a bit clearer if it were "Table key length 5123 is larger than max allowed".